sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/datasources/samples/SamplesDbConnector.groovy

```diff
@@ -35,8 +35,9 @@ class SamplesDbConnector implements CountSamplesDataSource, DownloadSamplesDataS
      * connect to the database
      * @since 1.0.0
      */
-    SamplesDbConnector(ConnectionProvider connectionProvider) {
+    SamplesDbConnector(ConnectionProvider connectionProvider, SampleTrackingService sampleTrackingService) {
```

Comment:
Why don't we use the sample tracking service directly but go over a database connector?